src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -261,7 +263,7 @@ export default {
 			filters: { },
 			targetedLoanChannelURL: null,
 			targetedLoanChannelID: null,
-			loanChannel: () => {},
+			loanChannel: {},
```

Comment:
This should not be a function and returns a weird value before its set while loading

Reply:
Wonder if this will affect reactivity when moving between categories...ie lbc to category to lbc and back to a different category. or even clicking into one of your exp categories...